packages/venia-concept/src/util/omit.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Return a copy of an object with one named property omitted.
+ * Short version of lodash.omit.
+ *
+ * Useful when a reducer needs to drop something from state, without using
+ * the `delete` keyword to mutate the existing state. The `delete` keyword
+ * is also a deopt.
+ *
+ * @param {object} obj -- Object to copy
+ * @param {string} omitKey -- Property name to omit
+ * @returns {object} Copy of the object with the specified key omitted.
+ *
+ * @example
+ *
+ * ```js
+ * const musketeers = {
+ *   athos: true,
+ *   porthos: true,
+ *   aramis: true
+ * };
+ *
+ * console.lot(omit(musketeers, 'athos'));
+ *
+ * {
+ *   porthos: true,
+ *   aramis: true
+ * }
+ */
+
+module.exports = (obj, omitKey) =>
```

Comment:
In general, James, are you sure that we need to add custom implementation of omit function to the project? My personal opinion, that it will be better to add lodash or underscore to the project. This libraries contains a lot of useful methods, that we will need in our app in the future like omit. If we decide to write our own implementations of such methods, than we will have to maintain our code, cover it with tests, write docs etc. Lodash is very popular library. The only thing we should remember is to avoid imports `import _ from 'lodash';` and just write, for example, `import omit from 'lodash/omit';`

Reply:
By the way, it will be useful to add ability to pass an array of omitKeys to the function.

Reply:
In my experience, Lodash adds a lot of file size, even when using tree-shaking. There are a few places in Venia and Peregrine where a Lodash method might be useful, but so far we have avoided its use. But I could be persuaded!

If you think that Lodash would ultimately help the codebase, then I would love for you to create a branch, refactor those parts of Venia and Peregrine to use Lodash, make sure they do not increase bundle size, and open a pull request!